### ISSUE TITLE
Display quoted message above the user message

### ISSUE THREAD

**Issue by Otiel** [RocketChat/Rocket.Chat#27102]

**Comment by engelgabriel** [comment]
@ggazzo can we do that with the new message parser?

**Comment by ankar84** [comment]
@engelgabriel @ggazzo it definitely make sense!! All that issue with many comments and reactions a good illustration of fact, community need that important improvement a lot!

**Comment by engelgabriel** [comment]
@ggazzo can you link the PR that fixed this here?

**Comment by ggazzo** [comment]
we will never have a pr who **fixes** this, because this is not a issue/bug. 

This is a design change, based on user input or feature/improvement request.

I will put the design team in the loop, if they believe this is the best option, I'm sure it will be implemented soon

btw this is something that @milton-rucks can help as well

**Comment by engelgabriel** [comment]
@dudanogueira did someone already work on this?

**Comment by milton-rucks** [comment]
I agree with the change. We are considering it for the next semester.

**Comment by dudanogueira** [comment]
> @dudanogueira did someone already work on this?

Oooops! For some reason I believed this was delivered in 5.4, as I mentioned at our monthly [Community Open Call ](https://open.rocket.chat/channel/Rocket-Chat-Open-Community)

But it was not. Sorry! But worry not! 

Now we have everything in place to deliver this and the dark mode ([that you can already preview with that cool Easter Egg](https://forums.rocket.chat/t/5-4-0-dark-theme/15504)) in 6.0
